ovs-p4rt/sidecar/testing/table_entry_test.h

```diff
@@ -42,6 +42,17 @@ class TableEntryTest : public ::testing::Test {
     }
   }
 
+  static uint16_t DecodePortValue(const std::string& string_value) {
```

Comment:
Abstracted the specific decoding algorithms for Port and VNI parameters, reducing repetition and making it easier to make changes to those algorithms if the need arises (as was the case with this PR).